corehq/apps/data_dictionary/static/data_dictionary/js/data_dictionary.js

```diff
@@ -180,6 +263,22 @@ hqDefine("data_dictionary/js/data_dictionary", [
             });
         };
 
+        subscribePropObservable(self.description);
+        subscribePropObservable(self.label);
+        subscribePropObservable(self.fhirResourcePropPath);
+        subscribePropObservable(self.dataType);
+        subscribePropObservable(self.deprecated);
+        subscribePropObservable(self.deleted);
+        subscribePropObservable(self.removeFHIRResourcePropertyPath);
+        self.allowedValues.on('change', changeSaveButton);
+        self.allowedValues.on('change', self.allowedValuesChanged);
+        self.hasChanges = true;
```

Comment:
The `hasChanges` property shouldn't default to `true`, as this would mean even existing properties with no changes will be marked as having changes.

Reply:
Interesting. That makes sense.

That line originated from [here when a new case property is created](https://github.com/dimagi/commcare-hq/blob/da37844851659e8e50b44061a7cee9ecff04493f/corehq/apps/data_dictionary/static/data_dictionary/js/data_dictionary.js#L491).

Removed in c9cbf76416e